contracts/modules/andromeda-schema/src/mock.rs

```diff
@@ -0,0 +1,77 @@
+#![cfg(all(not(target_arch = "wasm32"), feature = "testing"))]
+use crate::contract::{execute, instantiate, query};
+use andromeda_modules::schema::{ExecuteMsg, InstantiateMsg, QueryMsg, ValidateDataResponse};
+use andromeda_testing::mock::MockApp;
+use andromeda_testing::{
+    mock_ado,
+    mock_contract::{ExecuteResult, MockADO, MockContract},
+};
+use cosmwasm_std::{Addr, Coin, Empty};
+use cw_multi_test::{Contract, ContractWrapper};
+
+pub struct MockSchema(Addr);
+mock_ado!(MockSchema, ExecuteMsg, QueryMsg);
+
+impl MockSchema {
+    pub fn instantiate(
+        code_id: u64,
+        sender: Addr,
+        app: &mut MockApp,
+        kernel_address: String,
+        owner: Option<String>,
+        schema_json_string: String,
+    ) -> MockSchema {
+        let msg = mock_schema_instantiate_msg(kernel_address, owner, schema_json_string);
+        let addr = app
+            .instantiate_contract(
+                code_id,
+                sender.clone(),
+                &msg,
+                &[],
+                "Schema Contract",
+                Some(sender.to_string()),
+            )
+            .unwrap();
+        MockSchema(Addr::unchecked(addr))
+    }
+
+    pub fn execute_update_schema(
+        &self,
+        app: &mut MockApp,
+        sender: Addr,
+        funds: Option<Coin>,
+        new_schema_json_string: String,
+    ) -> ExecuteResult {
+        let msg = ExecuteMsg::UpdateSchema {
+            new_schema_json_string,
+        };
+        if let Some(funds) = funds {
+            app.execute_contract(sender, self.addr().clone(), &msg, &[funds])
```

Comment:
nit: conditionally build the `funds` vector and call this method once

Reply:
Sure, I resolved it.